### ISSUE TITLE
need Game Installation Folder Change or Games Backup/Restore option

### ISSUE THREAD

**Issue by SaifMahmud366** [nikita36078/J2ME-Loader#1068]
### Quick summary

1. I am unable to access the data folder on my Android 13 device to back up/ restore installed games and am requesting a change in the game installation folder to a location that is accessible (emulated 0 > J2Me Loader)
2. Or make a Game backup and restore system

### Details

I'm using Android 13. if i install any games it goes _Android > Data_ folder. you know that we can not access Data folder on updated Android version. If i want to make a backup of my installed games i can not do that. if you change the location of game installation folder like **emulated 0 > J2Me Loader** it will help a lot for backup of installed games for changing device. 
i have a backup file when i used Android 6 device, but i cant not set those backup files into my new device (Android 13)
**Please Change the Game installation Folder**
**Or make a Game backup and restore system**

**Comment by MrsEWE44** [comment]
look at me repository , is added your option.
https://github.com/MrsEWE44/J2ME-Loader

**Comment by SaifMahmud366** [comment]
> look at me repository , is added your option. https://github.com/MrsEWE44/J2ME-Loader

yeah its working...but, add this feature in official app..cs, **your app is not on play store**
